### PR TITLE
UWP 6.0.4 release notes

### PR DESCRIPTION
**Pull request by HollyAM**

### CODE REVIEW

File: releases/UWP/net-native1.7/README.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
-# .NET Native 1.7 (UWP 5.4.x) Release Notes
+# .NET native tools 1.7 Release Notes
 
 You can see what was included in each .NET Native 1.7 ([Microsoft.NETCore.UniversalWindowsPlatform 5.4.x](https://www.nuget.org/packages/Microsoft.NETCore.UniversalWindowsPlatform)) release, below.
 
 When using Visual Studio these packages require Visual Studio 2017 or later.
```
Comment (rpetrusha) [inline releases/UWP/net-native1.7/README.md]: Can they be used outside of Visual Studio? 
If not, just "These packages require..."
If so, "If used with Visual Studio, these packages..."
Same comment applies to line 5 of the next document.

File: releases/UWP/net-native1.7/README.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
-# .NET Native 1.7 (UWP 5.4.x) Release Notes
+# .NET native tools 1.7 Release Notes
 
 You can see what was included in each .NET Native 1.7 ([Microsoft.NETCore.UniversalWindowsPlatform 5.4.x](https://www.nuget.org/packages/Microsoft.NETCore.UniversalWindowsPlatform)) release, below.
```
Comment (rpetrusha) [inline releases/UWP/net-native1.7/README.md]: no comma before below.

File: releases/UWP/net-native1.7/README.md [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
 
 When using Visual Studio these packages require Visual Studio 2017 or later.
 
+### UWP 6.0.4 (.NET native tools 1.7.2) (December 4th, 2017)
+- Fixed an unhandled exception when ClientWebSocket loses connection to server. [518456]
+- Fixed a regression in the 1.7 toolchain causing winmd file corruption, when the file stream is left open. [496929]
+
 ### 5.4.1 (October 9, 2017)
 - Fixed an issue with Microsoft.NetNative.targets that prevented C++ hybrid apps from compiling; apps would fail to compile due to an attempt to write a file to a folder that does not exist. [511674]
```
Comment (rpetrusha) [inline releases/UWP/net-native1.7/README.md]: does not --> did not

File: releases/UWP/net-native2.0/README.md [hunk 4]
```diff
@@ -1,10 +1,17 @@
-# .NET Native 2.0 (UWP 6.0.x) Release Notes
+﻿# .NET native tools 2.0 (UWP 6.0.x) Release Notes
 
 You can see what was included in each .NET Native 2.0 ([Microsoft.NETCore.UniversalWindowsPlatform 6.0.x](https://www.nuget.org/packages/Microsoft.NETCore.UniversalWindowsPlatform)) release, below.
```
Comment (rpetrusha) [inline releases/UWP/net-native2.0/README.md]: no comma before below

File: releases/UWP/net-native1.7/README.md (hunk 3 above)
Comment (HollyAM) [reply releases/UWP/net-native1.7/README.md]: This PR isn't touching that section. I agree that probably should be resolved in another PR.

**Pull request merged**